udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -318,22 +356,21 @@ <h2 class="fs-xxl">Embed</h2>
         {{ hook('dataset.display.after-description') }}
     </div>
 </section>
-
 {% block community %}
-<section class="community_container bg-grey-400 text-white py-hg" id="community">
+<section class="community_container py-hg" id="community">
     <header>
         <div class="container">
             <div class="row">
-                <div class="col-3 col-md-12">
+                <div class="col-6 col-md-12">
                     <h2>{{ _('Community contributions') }}</h2>
```

Comment:
Do we still want this section since we now have a Summary?

Reply:
I'll remove it